src/modules/tests/candidate/candidate.selector.ts

```diff
@@ -20,3 +20,12 @@ export const formatDriverNumber = (driverNumber: string) => {
 };
 
 export const getCandidateEmailAddress = (candidate: Candidate) => candidate.emailAddress ? candidate.emailAddress : '';
+
+export const getPostalAddress = (candidate: Candidate) => candidate.candidateAddress;
+
+export const formatAddress = (address: Address) => {
+  const regex = new RegExp('[0-9]', 'g');
```

Comment:
this shouldn't be in the selector as it isn't selecting any data from the state

Reply:
fixed